README.md

```diff
@@ -124,6 +124,57 @@ Using JSON payload for constructing a message is also available:
     SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
 ```
 
+### Update the message
+
+Using `update-ts: ${{ steps.slack.outputs.ts }}` will update the message that was posted. 
```

Comment:
This sentence can be a bit more detailed. How about elaborating like this?

```suggestion
If you would like to notify the real-time updates on a build status, you can modify the message your build job posted in the subsequent steps. In order to do this, the steps after the first message posting can have `update-ts: ${{ steps.slack.outputs.ts }}` in their settings. With this, the step updates the already posted channel message instead of posting a new one.
```